### PR TITLE
Change to "cycling_speed' on input

### PR DESCRIPTION
**Pull request by dnesbitt61**
Validate input walking speed is within a reasonable range.

### CODE REVIEW

File: src/sif/pedestriancost.cc [hunk 1]
```diff
@@ -187,6 +192,13 @@ PedestrianCost::PedestrianCost(const boost::property_tree::ptree& pt)
   driveway_factor_   = pt.get<float>("driveway_factor", kDefaultDrivewayFactor);
   step_penalty_      = pt.get<float>("step_penalty", kDefaultStepPenalty);
 
+  // Validate speed (make sure it is in the accepted range)
+  if (walking_speed_ < kMinWalkingSpeed || walking_speed_ > kMaxWalkingSpeed) {
+    LOG_ERROR("Outside valid walking speed range " +
```
Comment (kevinkreiser) [inline src/sif/pedestriancost.cc]: shouldnt this be a warn?

**Comment by dgearhart** [comment]
:shipit:

File: src/sif/bicyclecost.cc [hunk 2]
```diff
@@ -296,7 +294,9 @@ BicycleCost::BicycleCost(const boost::property_tree::ptree& pt)
   // flat roads. If not present or outside the valid range use a default speed
   // based on the bicycle type.
   uint32_t t = static_cast<uint32_t>(bicycletype_);
-  speed_ = pt.get<float>("speed", kDefaultCyclingSpeed[t]);
+  speed_ = pt.get<float>("cycling_speed", kDefaultCyclingSpeed[t]);
+
+  // Validate speed (make sure it is in the accepted range)
   if (speed_ < kMinCyclingSpeed || speed_ > kMaxCyclingSpeed) {
     LOG_ERROR("Outside valid cycling speed range " + std::to_string(speed_) +
```
Comment (kevinkreiser) [inline src/sif/bicyclecost.cc]: Same here

**Comment by kevinkreiser** [comment]
i guess i was always thinking that if we recover from it, its a warn. but if we are erroring why arent we throwing an exception. maybe that is too strict?

**Comment by kevinkreiser** [comment]
otherwise fine of course

**Pull request merged**